FlowCrypt/Controllers/Inbox/InboxRenderable.swift

```diff
@@ -65,4 +63,37 @@ extension InboxRenderable {
         self.date = date ?? Date()
         self.wrappedType = .thread(thread)
     }
+
+    @MainActor func hideErrorLabel() {
+
+    }
+
+    private static func messageTitle(with thread: MessageThread, and folderPath: String) -> String {
+        let sentPath = "SENT"
+        guard let myEmail = DataService.shared.email else { return "" }
+
+        // for now its not exactly clear how titles on other folders should looks like
+        // so in scope of this PR we are applying this title presentation only for "sent" folder
+        if folderPath == sentPath {
+            var emails = thread.messages.compactMap(\.sender).unique()
+            // if we have only one email, it means that it could be "me" and we are not
+            // clearing our own email from that
+            if emails.count > 1 {
+                if let i = emails.firstIndex(of: myEmail) {
+                    emails.remove(at: i)
+                }
+            }
+            let receivers = emails
+                .compactMap { $0.components(separatedBy: "@").first ?? "" }
```

Comment:
`?? ""` isn't needed here, as `compactMap` will automatically delete all `nil` values

Reply:
It's not only not needed, but actively harmful - now we have to also handle empty strings, instead of only nils which get filetered with `compactMap`.